src/components/highlighted-places/index.tsx

```diff
@@ -25,7 +25,7 @@ const HighlightedPlaces = () => {
   const isHighlightedPlace = Object.values(HIGHLIGHTED_PLACES).includes(location_id);
 
   return (
-    <div className="flex justify-between space-x-6">
+    <div className="flex justify-between space-x-6 px-10">
```

Comment:
do we need both space-x and px-10?

Reply:
Yes, px-10 is for padding all around de div and space-x is for space between two elements inside the div